core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-daemonset.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Prepare configuration and upgrade/install Helm chart
+  vars:
+    # Handling "undefined", "null", "empty" and "boolean" values all at once.
+    disable_helm_chart_bool: "{{ specification.disable_helm_chart | default(false, true) | bool }}"
+
+  delegate_to: localhost
+  become: false
+  run_once: true
+  when: not disable_helm_chart_bool
+  block:
+    - name: Set Filebeat's Chart file name to install
+      set_fact:
+        exporter_chart_file_name: "{{ specification.files.filebeat_helm_chart_file_name }}"
+
+    - name: Download Filebeat's Chart File
+      include_role:
+        name: download
+        tasks_from: download_file
+      vars:
+        file_name: "{{ exporter_chart_file_name }}"
+        repository_url: http://localhost/epirepo
+
+    - name: Install Filebeat as DaemonSet using custom config file
+      block:
+        - name: Set Helm chart values from custom configuration
+          set_fact:
+            _helm_chart_values: "{{ lookup('template', '../templates/custom-chart-values.j2') }}"
```

Comment:
Do we need this lookup there?

Reply:
True, it can be done more easily. Done :)